paws_collector.js

```diff
@@ -96,14 +96,14 @@ class PawsCollector extends AlAwsCollector {
         }
     }
 
-    constructor(context, {aimsCreds, pawsCreds}, childVersion, healthChecks = [], statsChecks = []) {
+    constructor(context, {aimsCreds, pawsCreds}, childVersion,streams =[], healthChecks = [], statsChecks = []) {
```

Comment:
please exctract `paws_collector` lib changes into separate PR. Later on you'll need to update `paws-collector` dependency in collectors' package.json's

Reply:
Okay I will raise separate pr with latest version of al-aws-collector and bump the paws version.